public-api/db-cdr/generate-cdr/move-aggregated-counts.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# This moves aggregated count data csvs from prod bucket to public bucket to be imported to cloudsql
```

Comment:
Is public being used here to indicate an actual public ACL? Or is this using public in the way that databrowser normally uses public (e.g. as a database name).

In practice, what is the destination project used for --public-project ?

Reply:
hmmm, it means public aka "test" bucket/ project.

Reply:
OK. Should it be called "test-bucket" perhaps? Or something else? I'm just getting confused as to whether this means the ACL on this bucket actually public - is it?

Reply:
Ch, changed the wording to be test-bucket. The bucket is not public to all users though. It has access to few service accounts and our team.